chroma_core/models/client_mount.py

```diff
@@ -18,7 +19,12 @@
 
 class LustreClientMount(DeletableStatefulObject):
     host = models.ForeignKey("ManagedHost", help_text="Mount host", related_name="client_mounts", on_delete=CASCADE)
-    filesystem = models.ForeignKey("ManagedFilesystem", help_text="Mounted filesystem", on_delete=CASCADE)
+    filesystem = models.CharField(
+        max_length=ManagedFilesystem._meta.get_field("name").max_length,
```

Comment:
```suggestion
        max_length=8,
```

It's better to just hardcode this, I don't think it will ever change.